ports/unix/stack_size.h

```diff
@@ -0,0 +1,29 @@
+#ifndef MICROPY_INCLUDED_UNIX_STACK_SIZE_H
```

Comment:
Please add a copyright and license header.

Reply:
Oops, rebased and added!